lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -294,7 +299,7 @@ defmodule ExUnit.Case do
 
   @type env :: module() | Macro.Env.t()
   @compile {:no_warn_undefined, [IEx.Pry]}
-  @reserved [:module, :file, :line, :test, :async, :registered, :describe]
+  @reserved [:module, :file, :line, :test, :async, :group, :registered, :describe]
```

Comment:
Is this list missing `:parameterize`?

Reply:
I don't think it ever makes to the tags, does it?

Reply:
Oh you're right. I checked and we also don't need to add `:group` to the list of reserved tags, not unless we want to add it to the tags - so I've removed it in https://github.com/elixir-lang/elixir/pull/13897/commits/55f38a432395e9fede81b8ebf7052ee649964c4e.